client/app/lib/components/ProviderWrapper.jsx

```diff
@@ -24,13 +32,26 @@ const propTypes = {
   children: PropTypes.element.isRequired,
 };
 
+const tailwindConfig = resolveConfig(tailwindUserConfig);
+
+const pxInInt = (pixels) => parseInt(pixels.replace('px', ''), 10);
+
 const themeSettings = {
   palette,
   // https://material-ui.com/customization/themes/#typography---html-font-size
   // https://material-ui.com/style/typography/#migration-to-typography-v2
   typography: {
     htmlFontSize: 10,
   },
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: pxInInt(tailwindConfig.theme.screens.sm),
```

Comment:
Besides these, are there any other theme settings to be reconciled between mui and tailwind?

Reply:
See my comment below for number 4.